utilities/validator.py

```diff
@@ -39,6 +18,12 @@ def verify_calendar_id(calendar_id):
         return False
 
 
+def verify_location_id(kwargs):
```

Comment:
can we take care of soft delete in this function please

Reply:
On it @michael-basweti